prov/gni/test/cntr.c

```diff
@@ -98,6 +98,7 @@ static inline void cntr_setup_eps(void)
 	hints->mode = ~0;
 
 	hints->fabric_attr->name = strdup("gni");
+	hints->caps = GNIX_EP_RDM_PRIMARY_CAPS;
```

Comment:
I can see that this could cause some confusion but I left it alone for now.  I don't think it's too unreasonable that we hang when the receiver is not progressing libfabric and has auto progress disabled.